### ISSUE TITLE
The Emoji issue still persist

### ISSUE THREAD

**Issue by m-primo** [FlorianWoelki/obsidian-iconize#468]
**Describe the bug**

Same as 
#438 #458 

**To Reproduce**
1. Just install & enable the plugin

**Screenshots**

With iconize disabled:
![image](https://github.com/FlorianWoelki/obsidian-iconize/assets/44984918/24033b0b-c429-4c87-8c8f-6c07e1b51a82)

With iconize enabled:
![image](https://github.com/FlorianWoelki/obsidian-iconize/assets/44984918/071cf302-bf5c-4cec-b3a3-11c633f3750d)

**Additional context**
For more details follow the same old issues that mentioned above.
I don't know what did you update in version 2.11.9 that still till now not fixed. But for me I'll just download the `main.js` file of the latest working version and install it manually, and I wouldn't care about other updates.

**Comment by FlorianWoelki** [comment]
hey, I am not able to reproduce this issue and seems like others cannot reproduce this one as well. 
maybe you can clarify a little bit more when this issue actually occurs so that I can reproduce it.

**Comment by pyxelr** [comment]
I also confirm that I cannot reproduce this bug in neither text mode (source mode, live preview or reading). I am using Iconize `2.11.13` in Obsidian `1.5.12`.

**Comment by paxnovem** [comment]
I am currently still experiencing this issue as well with the Tasks plugin emoji format in the editor (not in the dataview queries). The issue goes away if I disable iconize.
Obsidian: 1.5.12
Iconize: 2.11.13
Tasks: 7.0.0
Theme: Minimal 7.5.4
Experiencing this on android 13 and Windows 10.

**Comment by m-primo** [comment]
> hey, I am not able to reproduce this issue and seems like others cannot reproduce this one as well. maybe you can clarify a little bit more when this issue actually occurs so that I can reproduce it.

As I stated, the issue still persists, and to reproduce, just install & enable the plugin, that's the way I can reproduce it.
I'm using Windows 11.
Obsidian version 1.5.12.
Please let me know if you need specific information.

**Comment by ghostneverdie** [comment]
macOS 13.2 (22D49)   obsidian1.5.11 have the same bug,all the emoji cann`t see.

**Comment by FlorianWoelki** [comment]
> > hey, I am not able to reproduce this issue and seems like others cannot reproduce this one as well. maybe you can clarify a little bit more when this issue actually occurs so that I can reproduce it.
> 
> As I stated, the issue still persists, and to reproduce, just install & enable the plugin, that's the way I can reproduce it. I'm using Windows 11. Obsidian version 1.5.12. Please let me know if you need specific information.

could you give me an excerpt of your `data.json` file?

do you use native emojis or twemoji? please also check the settings in Iconize (if the emoji mode is set to `None`, set it to your preferred option)

**Comment by paxnovem** [comment]
I think the behavior is it being set to none combined with it appearing when on the line. I don't believe I ever set that setting (I mostly use folder icons, but maybe hit it at some point). Is none the intended default behavior?

**Comment by FlorianWoelki** [comment]
> I think the behavior is it being set to none combined with it appearing when on the line. I don't believe I ever set that setting (I mostly use folder icons, but maybe hit it at some point). Is none the intended default behavior?

yes, it broke some thing for users lately. the default behaviour is now that the emoji mode will be set to "native" whenever you install Iconize. but, this is only since version 2.11.10 (I think).

**Comment by nicopico-dev** [comment]
I fixed the issue on my side by disabling the "Toggle icons while editing notes" in Iconize. I only use Iconize for folders and files.

<img width="763" alt="image" src="https://github.com/FlorianWoelki/obsidian-iconize/assets/1057061/a9bb95a5-31f9-4530-9cd8-11748f9ef700">

**Comment by m-primo** [comment]
> could you give me an excerpt of your `data.json` file?

[data.json](https://github.com/FlorianWoelki/obsidian-iconize/files/15083664/data.json)

> do you use native emojis or twemoji?

`none`

> please also check the settings in Iconize (if the emoji mode is set to `None`, set it to your preferred option)

I have changed it to `Native` now, for now everything seem to work, I haven't tried it with all plugins.

I'll close this issue for now, and if anything comes up, I'll either open a new issue, or just go back to v2.11.5 or smth.

Thanks.

**Comment by m-primo** [comment]
@nicopico-dev 

> I fixed the issue on my side by disabling the "Toggle icons while editing notes" in Iconize. I only use Iconize for folders and files.

Thanks for this, I did disable it too, since I also use it for folders only, idk why they added it as a global icon plugin! It should be just for folders as it was before, like the name `icon-folder`, things got bad after that change.

**Comment by mrcarriere** [comment]
Chiming in, as I found this when trying to deal with the issue (fwiw, came from [here](https://forum.obsidian.md/t/emojis-no-longer-show-up-in-edit-mode/80128/4)):

* Changing the Emoji Style from `None` to `Native` resolved the issue for me. My settings were previously at `None` for some reason.
* The other workaround did work too (disabling `Toggle icons while editing notes`) but the first workaround solves this without needing to disable this feature.
